geoapi/routes/projects.py

```diff
@@ -208,9 +208,15 @@ def get(self, projectId: int):
     @project_admin_or_creator_permissions
     def delete(self, projectId: int):
         u = request.current_user
-        logger.info("Delete project:{} for user:{}".format(projectId,
-                                                           u.username))
-        return ProjectsService.delete(db_session, u, projectId)
+        # Retrieve the project using the projectId to get its UUID
+        project = ProjectsService.get(db_session, project_id=projectId, user=u)
+        # Check if the project exists and log the information including the UUID
+        if project:
+            logger.info("Delete project:{} with UUID:{} for user:{}".format(
```

Comment:
instead of `UUID` what about `project_uuid`?  

There are lots of uuids in the code for things like task, streetview, asset path etc. so maybe being more specific would improve things.

I got the somewhat related PR https://github.com/TACC-Cloud/geoapi/pull/161/ where I did something different but I am going to alter that to match what we decide here and the rest of the code. 

Seems like the rest of the log statements use `project:` when referring to the int id 👍  so just what should we call the project uuid.

Reply:
I can change it to "Project UUID:" or "project_uuid:"